lib/mercadopago/client.rb

```diff
@@ -151,7 +152,7 @@ def notification_preapproval(preapproval_id)
     # - search_hash: the search hash to find collections.
     #
     def search(search_hash)
-      MercadoPago::Collection.search(@access_token, search_hash, @sandbox)
+      MercadoPago::Collection.search(@access_token, search_hash, @sandbox, @auto_paginate)
```

Comment:
@cleicar I'm not sure if `@auto_paginate` is defined here, could you use `auto_paginate` instead?